src/containers/RecordingsTable.tsx

```diff
@@ -1,24 +1,23 @@
 import { CircularProgress } from '@material-ui/core';
-import React, { FunctionComponent, useContext, useEffect, useMemo } from 'react';
-import { Link } from 'react-router-dom';
-import { getRecordingInfo } from '../actions/getRecordingInfo';
+import React, { FunctionComponent, useCallback, useMemo } from 'react';
+import { WorkspaceInfo } from '../AppContainer';
 import NiceTable from '../components/NiceTable';
-import { HitherContext } from '../extensions/common/hither';
-import { getPathQuery } from '../kachery';
+import { useRecordingInfos } from '../extensions/common/getRecordingInfo';
 import { Recording, RecordingInfo } from '../reducers/recordings';
-import { Sorting } from '../reducers/sortings';
-import { WorkspaceInfo } from '../reducers/workspaceInfo';
+import { Sorting, SortingInfo } from '../reducers/sortings';
+import { WorkspaceRouteDispatch } from './WorkspaceView';
+import './WorkspaceView.css';
 
 interface Props {
     workspaceInfo: WorkspaceInfo
     recordings: Recording[]
     sortings: Sorting[]
     onDeleteRecordings: (recordingIds: string[]) => void
-    onSetRecordingInfo: (a: { recordingId: string, recordingInfo: RecordingInfo }) => void
+    workspaceRouteDispatch: WorkspaceRouteDispatch
```

Comment:
This is the hook/passed-in utility that exposes the ability to redirect the page location, right?

Reply:
right